README.rst

```diff
@@ -34,6 +34,27 @@ One Time Setup
   # Here's how you might do that if you have virtualenvwrapper setup.
   mkvirtualenv -p python3.8 learning-assistant
 
+In your ``private.txt`` requirements file in edx-platform, add:
+
+.. code-block::
+
+  -e /edx/src/learning-assistant
+
+In your ``private.py`` settings file in edx-platform, add the below settings. The value of the API key shouldn't matter, because it's not being used at this point, but the setting needs to be there.
+
+.. code-block::
+
+  CHAT_COMPLETION_API = ''
+  CHAT_COMPLETION_API_KEY = ''
```

Comment:
This also needs a value, although it's value doesn't matter at this point.

Reply:
Ah makes sense! Added a comment to code block.